.github/workflows/test.yml

```diff
@@ -145,4 +145,9 @@ jobs:
         go get github.com/onsi/gomega/...
         ginkgo --focus "${TEST_NAME}" -nodes=3 -x test/e2e/
 
-# TODO if int test failed - stop e2e, add job for cleanup Atlas projects/clusters
+    - name: Upload operator logs
+      if: ${{ failure() }}
+      uses: actions/upload-artifact@v2
```

Comment:
Looks like an amazing feature! 🥇 
Where does it save the files? Somewhere internally on Github and then shows in the UI?

Reply:
![image](https://user-images.githubusercontent.com/5437289/110111043-c0fe0d80-7dc0-11eb-9ffd-b01306c9f5d2.png)
in the summary field